### PR TITLE
ui: clusterviz: move clusterviz to overview page

### PR DESCRIPTION
**Pull request by vilterp**
screen shot 2018-02-15 at 4 22 32 pm

### CODE REVIEW

File: pkg/ui/ccl/src/views/clusterviz/containers/map/localityView.tsx [hunk 1]
```diff
@@ -38,7 +38,8 @@ const LIVENESS_TOTAL_WIDTH = LIVENESS_ICON_RADIUS * 2 + LIVENESS_ICON_SPACING;
 class LocalityView extends React.Component<LocalityViewProps & WithRouterProps> {
   onClick = () => {
     const localityTree = this.props.localityTree;
-    const destination = CLUSTERVIZ_ROOT + "/" + generateLocalityRoute(localityTree.tiers);
+    const destination = CLUSTERVIZ_ROOT + generateLocalityRoute(localityTree.tiers);
+    console.log(destination);
```
Comment (couchand) [inline pkg/ui/ccl/src/views/clusterviz/containers/map/localityView.tsx]: beep

**Comment by mrtracy** [comment]
:lgtm:, although obviously there's some debt accumulating here.

---

Review status: 0 of 14 files reviewed at latest revision, 1 unresolved discussion, some commit checks pending.

---

*[pkg/ui/ccl/src/views/clusterviz/containers/map/nodeList.tsx, line 22 at r3](https://reviewable.io:443/reviews/cockroachdb/cockroach/22763#-L5QZmEBoS2JdQdVcuNI:-L5QZmEBoS2JdQdVcuNJ:bubxysn) ([raw file](https://github.com/cockroachdb/cockroach/blob/68cf99f04708bd6d49a4ff6a491dc1910a52aeb2/pkg/ui/ccl/src/views/clusterviz/containers/map/nodeList.tsx#L22)):*
> ```XML
> export default class NodeList extends React.Component<RouterState & { router: InjectedRouter }> {
>   handleMapTableToggle = (opt: DropdownOption) => {
>     this.props.router.push(`/overview/${opt.value}`);
> ```

Can you at least add a TODO to refactor this copy/paste between the nodesoverview and clusterviz?  The navigation bar has a bunch of common elements, it should be a component.

It seems like the right move for now to just get it committed like this, but lets at least acknowledge the obvious stuff.

Or at least, create a 2.1 issue to clean up this debt.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/cockroach/22763#-:-L5Q_qa2uX06bOYbBjiy:brzwlnz)*

**Review by couchand — APPROVED** [review]
:shipit:

**Comment by couchand** [comment]
> obviously there's some debt accumulating here

😜

**Comment by vilterp** [comment]
@mrtracy Added a TODO do dedup those, plus a card in the Cluster Viz project.

**Comment by couchand** [comment]
@vilterp I just noticed that the first commit probably should have had a release note attached!

**Pull request merged**